src/components/button/Button.js

```diff
@@ -21,7 +22,8 @@ const Button = (props) => {
       className={classNames(
         styles.button,
         styles[type] || styles.primary,
-        styles[size] || styles.default,
+        styles[size] || styles.medium,
```

Comment:
I don't think the `or` is necessary here, we have a default value and we have a validation to only accept certain values. Therefore `size` will always have a valid value. 

Please remove all these `||`

Reply:
I thought the same when I added the ones related to `shape`. Actually I wanted to remove them, but I was not sure... I'll delete it tomorrow!